playbooks/libvirt/openshift-cluster/tasks/configure_libvirt_network.yml

```diff
@@ -0,0 +1,27 @@
+---
+- name: Test if libvirt network for openshift already exists
+  command: "virsh -c {{ libvirt_uri }} net-info {{ libvirt_network }}"
+  register: net_info_result
+  changed_when: False
+  failed_when: "net_info_result.rc != 0 and 'error: Network not found:' not in net_info_result.stderr"
```

Comment:
This test fails when a locale is set.
Unfortunately, replacing
`command: "virsh -c {{ libvirt_uri }} net-info {{ libvirt_network }}"`
by:
`shell: "LANG=POSIX virsh -c {{ libvirt_uri }} net-info {{ libvirt_network }}"`
does **not** help.
See https://github.com/detiber/openshift-ansible/pull/20 for a solution proposal